### DISCUSSION TITLE
JupyterNB Interface - Single Cell for Variable Input

### DISCUSSION THREAD

**Discussion by rajavishah** [bilayer-containers/bilayers#24] [Q&A]
According to the discussion on Slack, Instead of having a bunch of cells or widgets for input parameters in Jupyter Notebook Interface, we would want to go with single cell. The approach should be to have a single cell that indicates the variables to be set, followed by a `!shell command` to execute. This would simplify the interface and avoid the need for multiple cells or widgets, especially when dealing with a large number of parameters like cellpose has

**Comment by gnodar01** [comment]
I believe this is now implemented, reopen if I'm incorrect.
